fastq_filtrator.py

```diff
@@ -0,0 +1,98 @@
+
+# ### Функция для прочтения файла по 4 строки:
+
+def read_4lines(file):
+    lines = []
+    for i in range(4):
+        line = file.readline()
+        if line == "":
+            return []
+        lines.append(line.replace("\n", ""))
+    return lines
+
+
+
+# ###  Функция для проверки GC-состава:
+
+def gc_bounds_func(inter_gc, lines):
+    cnt_gc = 0
```

Comment:
Это работает, но лучше использовать логические типы для таких штук
```suggestion
    cnt_gc = False
```